c++/src/kj/compat/http-test.c++

```diff
@@ -2307,6 +2307,10 @@ const byte WEBSOCKET_TWO_DEFLATE_BLOCKS_MESSAGE[] =
     {  0xc1, 0x0d, 0xf2, 0x48, 0x05, 0x00, 0x00, 0x00, 0xff, 0xff, 0xca, 0xc9, 0xc9, 0x07, 0x00 };
 // See this example: https://datatracker.ietf.org/doc/html/rfc7692#section-7.2.3.5
 // This uses two DEFLATE blocks in a single message.
+const byte WEBSOCKET_EMPTY_COMPRESSED_MESSAGE[] =
+    { 0xc1, 0x00 };
+const byte WEBSOCKET_EMPTY_SEND_COMPRESSED_MESSAGE[] =
+    { 0xc1, 0x80, 12, 34, 56, 78 };
```

Comment:
With the other comment in mind (regarding RFC), I think by adding an empty deflate block `0x00`, these would become:

```
const byte WEBSOCKET_EMPTY_COMPRESSED_MESSAGE[] =
    { 0xc1, 0x01, 0x00 };
const byte WEBSOCKET_EMPTY_SEND_COMPRESSED_MESSAGE[] =
    { 0xc1, 0x81, 12, 34, 56, 78, 0x00^12 };
```